virttest/defaults.py

```diff
@@ -18,7 +18,7 @@ def get_default_guest_os_info():
     Gets the default asset and variant information
     TODO: Check for the ARCH and choose corresponding default asset
     """
-    return {'asset': 'jeos-21-64', 'variant': 'JeOS.21'}
+    return {'asset': 'jeos-23-64', 'variant': 'JeOS.23'}
```

Comment:
I also dislike it, but as I said before, it's a current hosting limitation. I'm working on a better hosting though.